src/main/resources/hibernate.cfg.xml

```diff
@@ -48,7 +48,7 @@ Contributor(s):
         <property name="hibernate.connection.username">root</property>
         <property name="hibernate.connection.password">root</property>
 
-        <property name="hbm2ddl.auto">update</property>
+        <property name="hbm2ddl.auto">create</property>
```

Comment:
@HeriklesVinicyus Por que você voltou com o `create` na configuração do `hbm2ddl.auto`? Veja a issue #88.

Reply:
como eu mudei o método de Contagem do dos id's estava dando conflito com o que eu já tinha, quando mudei de update para create funcionou

Reply:
@HeriklesVinicyus Que conflito era esse? Qual o erro? Você experimentou apagar o BD na mão? @jjunio01 O que você acha dessa mudança?

Reply:
@fabriciofx, essa mudança irá recair no mesmo problema relatado na issue #88 . Se for retornado para create agora, em algum momento futuro teremos que mudar novamente para update.

Reply:
@jjunio01 Estou de acordo. @HeriklesVinicyus Veja se consegue fazer funcionar com o `update` ao invés do `create`. Uma vez conseguido, atualize este PR, ok?